modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/Checkpointer.java

```diff
@@ -407,8 +408,8 @@ void doCheckpoint() throws IgniteInternalCheckedException {
 
             if (chp.hasDelta()) {
                 if (log.isInfoEnabled()) {
-                    float totalDurationInSeconds = tracker.checkpointDuration(MILLISECONDS) / 1000.0f;
-                    float avgWriteSpeedInBytes = ((long) pageSize * chp.dirtyPagesSize) / totalDurationInSeconds;
+                    long totalDurationInSeconds = tracker.checkpointDuration(SECONDS);
```

Comment:
Nooo, you should have used microseconds, you made the calculation less precise now. Can you please improve it?

Reply:
> which will be equal to 7.81 megabytes per second, looks wrong

How is this wrong if it's right?

Reply:
Fix it.